### PR TITLE
Feat/sc 21 design updates

### PR DESCRIPTION
**Pull request by davidgilbertson**
Jira: https://govausites.atlassian.net/browse/SC-21

### CODE REVIEW

File: scripts/convert.js [hunk 1]
```diff
@@ -23,7 +23,6 @@ getStdin().then((rawCsv) => {
 
       if (!city) throw new Error(`${row.Cities} is not a known smart city`);
 
-      city.name = row.Cities;
```
Comment (davidgilbertson) [inline scripts/convert.js:26]: We ignore the spreadsheet version and just use the city object name.

**Comment by micapam** [comment]
maybe get rid of shortName from CityOverview.test.js as it's been removed everywhere else

**Review by micapam — APPROVED** [review]
one tiny comment, lgtm

**Comment by davidgilbertson** [comment]
Nicely spotted. Fixed

**Pull request merged**